### PR TITLE
Add support for other minecraft versions. (1.7.2, 1.7.10, 1.8)

### PR DESCRIPTION
**Pull request by mattmess1221**
Implements request from #3

### CODE REVIEW

**Comment by stephan-gh** [comment]
Thanks! You need to add custom JSON dependency configurations like https://github.com/Minecrell/VanillaGradle/blob/master/src/main/resources/versions/1.8.json so it doesn't add the whole FML dependencies to the IDE

**Comment by mattmess1221** [comment]
I've added the jsons.

File: src/main/resources/versions/1.7.2.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+            },
+            "extract": {
+                "exclude": [
+                    "META-INF/"
+                ]
+            }
+        }
+    ],
+    "mainClass": "net.minecraft.launchwrapper.Launch",
+    "minimumLauncherVersion": 9ff
```
Comment (stephan-gh) [inline src/main/resources/versions/1.7.2.json]: Is the rest of the JSON missing here? :o

Comment (mattmess1221) [inline src/main/resources/versions/1.7.2.json]: Oops. I must have accidentally overwritten the last part. It should just be a `}`

**Comment by mattmess1221** [comment]
Json's fixed.

**Comment by stephan-gh** [comment]
Have you tested that this is working properly now?

**Comment by mattmess1221** [comment]
I'm working on that now.

**Comment by stephan-gh** [comment]
It would be good if you could test the branch I've squashed already: https://github.com/Minecrell/VanillaGradle/commits/multiversion

**Comment by mattmess1221** [comment]
I still had some errors in the jsons, so this commit should fix it.

**Comment by mattmess1221** [comment]
btw, I know how to use buildSrc.

**Comment by stephan-gh** [comment]
buildSrc?

**Comment by mattmess1221** [comment]
If you create a folder in your build named buildSrc, gradle will build it
and add it to your build script classpath.

**Comment by stephan-gh** [comment]
Oh that's pretty neat, never noticed that

**Comment by stephan-gh** [comment]
Is the latest version of your pull request working properly now?

**Comment by mattmess1221** [comment]
Yes.

**Comment by stephan-gh** [comment]
Thanks!

**Pull request closed without merging**